source/documentation/deploying_services/opensearch.md

```diff
@@ -0,0 +1,281 @@
+
+```
+cf delete-service my-opensearch-service
+```
+
+Enter `yes` when asked for confirmation.
+
+<h2 id="maintaining-the-service">Maintaining the service</h2>
+
+### Data classification
+
+You can store data classified up to Official on the GOV.UK PaaS. Refer to the [data security classification documentation](/deploying_services/#data-security-classification) for more information.
+
+### OpenSearch backups
+
+Aiven automatically backs up all data stored within any OpenSearch service you create.
+
+Backups are taken every 2 hours. Data is retained for:
+
+- 2 days if you have a `tiny` plan
+- 14 days if you have a `small`, `medium` or `large` plan
+
+To restore data to an earlier state, you can visit the [GOV.UK PaaS support page](https://admin.london.cloud.service.gov.uk/support) or contact us at [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
+
+### View your OpenSearch data using Kibana
+
+Aiven provide a Kibana user interface through which you can explore the data in your OpenSearch. You can access Kibana using the following instructions:
+
+1. Run the following to get some valid credentials for your OpenSearch:
+
+    ```sh
+    cf create-service-key NAME_OF_YOUR_OPENSEARCH creds-for-kibana
+
+    cf service-key NAME_OF_YOUR_OPENSEARCH creds-for-kibana
+    ```
+
+    That outputs a hostname, username and password that you will use later.
+
+2. Set up an SSH tunnel to your target OpenSearch:
+
+    ```sh
+    cf ssh -L 4430:HOSTNAME_FROM_STEP_1:443 NAME_OF_AN_APP_YOU_CAN_SSH_TO
+    ```
+
+   OpenSearch instances are only accessible from the PaaS. This sets up a tunnel so that you can get access, similar to how we use [Conduit](https://github.com/alphagov/paas-cf-conduit) elsewhere.
+
+3. Access Kibana via your browser by going to [https://localhost:4430](https://localhost:4430).
+   When prompted for credentials, provide the username and password from step 1.
+   If you see a blank page then make sure your browser url is using `https` and not `http`.
+
+4. To clean up when you are done, `CTRL+C` the `cf ssh` session from step 2 and run `cf delete-service-key NAME_OF_YOUR_OPENSEARCH creds-for-kibana`.
+
+### IP Restrictions for OpenSearch
+
+You can add or remove IP restrictions for an OpenSearch service instance when:
+
+- creating a new service instance
+- updating an existing service instance
+
+### Add IP restrictions when creating a service instance
+
+You [create a new OpenSearch service instance](#set-up-the-service) by running `cf create-service`. You can enable optional IP restrictions in this new service instance by running:
+
+```
+cf create-service SERVICE_NAME -c '{"ip_filter": "IPADDRESS_1,IPADDRESS_2"}'
+```
+
+where:
+
+- `SERVICE_NAME` is a unique descriptive name for this service instance
+- `IPADDRESS_1...N` are the IP addresses you wish to add to the new service
+
+#### Add IP restrictions on an existing service instance
```

Comment:
Do we know how IP restrictions affect Kibana?

Reply:
Standard response here of "this is a copypasta from ES docs, are we updating it individually here and leaving the ES docs bad and dangly?"

Reply:
@cmcnallygds currently we don't, this story is really just meant to be a quick "copy the elasticsearch docs verbatim, so we can get OpenSearch announced ASAP so our tenants have as much time as possible to migrate before their existing ElasticSearch instances are yanked out from underneath them" one